.github/workflows/downstream.yaml

```
@@ -47,3 +47,20 @@ jobs:
         run: ./.kokoro/presubmit/common_test.sh
       - name: Perform downstream compatibility testing
         run: REPOS_UNDER_TEST="${{ matrix.repo }}" ./.kokoro/presubmit/downstream-compatibility.sh
+  downstream-compatibility-spring:
```

Please update this to reflect that the scope is limited to Spring Cloud GCP generator.
```suggestion
  downstream-compatibility-spring-generator:
```